cmd/offline/offline.go

```diff
@@ -70,6 +63,29 @@ func SaveHeartbeats(v *viper.Viper, heartbeats []heartbeat.Heartbeat) error {
 	return nil
 }
 
+func loadParams(v *viper.Viper) (paramscmd.Params, error) {
+	paramHeartbeat, err := paramscmd.LoadHeartbeatParams(v)
```

Comment:
When only saving extra heartbeats over the max, we don't want to load heartbeat params. That's the only case where we only want to save the given heartbeats without saving the param heartbeat.